<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>539</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="saveAsMenu">
     <property name="title">
      <string>Save &amp;As</string>
     </property>
    </widget>
    <addaction name="openAct"/>
    <addaction name="saveAsMenu"/>
    <addaction name="printAct"/>
    <addaction name="separator"/>
    <addaction name="exitAct"/>
   </widget>
   <widget class="QMenu" name="optionMenu">
    <property name="title">
     <string>&amp;Option</string>
    </property>
    <addaction name="penColorAct"/>
    <addaction name="brushColorAct"/>
    <addaction name="separator"/>
    <addaction name="clearScreenAct"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAct"/>
    <addaction name="aboutQtAct"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="undoAct"/>
    <addaction name="redoAct"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="optionMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QToolBar" name="drawToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="drawPencilAct"/>
   <addaction name="drawLineAct"/>
   <addaction name="drawRectAct"/>
   <addaction name="drawRoundRectAct"/>
   <addaction name="drawElliAct"/>
   <addaction name="eraseAct"/>
   <addaction name="drawTextAct"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContent">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="penStyleLayout">
       <item>
        <widget class="QLabel" name="penStyleLabel">
         <property name="text">
          <string>LineStyle</string>
         </property>
         <property name="buddy">
          <cstring>penStyleComboBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="penStyleComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="brushStyleLayout">
       <item>
        <widget class="QLabel" name="brushStyleLabel">
         <property name="text">
          <string>BrushStyle</string>
         </property>
         <property name="buddy">
          <cstring>brushStyleComboBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="brushStyleComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="penWidthLayout">
       <item>
        <widget class="QLabel" name="penWidthLabel">
         <property name="text">
          <string>Width</string>
         </property>
         <property name="buddy">
          <cstring>penWidthSlider</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="penWidthSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="penWidthNumber">
         <property name="text">
          <string>WIDTH</string>
         </property>
         <property name="buddy">
          <cstring>penWidthSlider</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="selectAct"/>
   <addaction name="cutAct"/>
   <addaction name="copyAct"/>
   <addaction name="pasteAct"/>
  </widget>
  <action name="aboutAct">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="aboutQtAct">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="penColorAct">
   <property name="text">
    <string>&amp;Pen Color...</string>
   </property>
  </action>
  <action name="penWidthAct">
   <property name="text">
    <string>Pen &amp;Width...</string>
   </property>
  </action>
  <action name="clearScreenAct">
   <property name="text">
    <string>&amp;Clear Screen</string>
   </property>
  </action>
  <action name="undoAct">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="redoAct">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="openAct">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="printAct">
   <property name="text">
    <string>&amp;Print...</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="drawPencilAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="scribble.qrc">
     <normaloff>:/icons/pencil.gif</normaloff>:/icons/pencil.gif</iconset>
   </property>
   <property name="text">
    <string>Pencil</string>
   </property>
   <property name="toolTip">
    <string>Draw arbitrary</string>
   </property>
  </action>
  <action name="drawLineAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="scribble.qrc">
     <normaloff>:/icons/line.gif</normaloff>:/icons/line.gif</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="toolTip">
    <string>Draw a line</string>
   </property>
  </action>
  <action name="drawRectAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="scribble.qrc">
     <normaloff>:/icons/rect.gif</normaloff>:/icons/rect.gif</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="toolTip">
    <string>Draw a rectangle</string>
   </property>
  </action>
  <action name="drawRoundRectAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="scribble.qrc">
     <normaloff>:/icons/roundrect.gif</normaloff>:/icons/roundrect.gif</iconset>
   </property>
   <property name="text">
    <string>Rounded Rectangle</string>
   </property>
   <property name="toolTip">
    <string>Draw a rounded rectangle</string>
   </property>
  </action>
  <action name="drawElliAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="scribble.qrc">
     <normaloff>:/icons/ellipse.gif</normaloff>:/icons/ellipse.gif</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
   <property name="toolTip">
    <string>Draw an ellipse</string>
   </property>
  </action>
  <action name="brushColorAct">
   <property name="text">
    <string>&amp;Brush Color...</string>
   </property>
  </action>
  <action name="eraseAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="scribble.qrc">
     <normaloff>:/icons/eraser.gif</normaloff>:/icons/eraser.gif</iconset>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
   <property name="toolTip">
    <string>Eraser</string>
   </property>
  </action>
  <action name="drawPolyAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="scribble.qrc">
     <normaloff>:/icons/polygon.gif</normaloff>:/icons/polygon.gif</iconset>
   </property>
   <property name="text">
    <string>Polygon</string>
   </property>
   <property name="toolTip">
    <string>Draw a polygon (right-click done)</string>
   </property>
  </action>
  <action name="drawTextAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="scribble.qrc">
     <normaloff>:/icons/text.gif</normaloff>:/icons/text.gif</iconset>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
   <property name="toolTip">
    <string>Put a text</string>
   </property>
  </action>
  <action name="drawCurveAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="scribble.qrc">
     <normaloff>:/icons/curve.gif</normaloff>:/icons/curve.gif</iconset>
   </property>
   <property name="text">
    <string>Curve</string>
   </property>
   <property name="toolTip">
    <string>Draw a curve</string>
   </property>
  </action>
  <action name="selectAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="scribble.qrc">
     <normaloff>:/icons/select.gif</normaloff>:/icons/select.gif</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>Select</string>
   </property>
  </action>
  <action name="drawPieAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="scribble.qrc">
     <normaloff>:/icons/pie.gif</normaloff>:/icons/pie.gif</iconset>
   </property>
   <property name="text">
    <string>Pie</string>
   </property>
   <property name="toolTip">
    <string>Draw a pie</string>
   </property>
  </action>
  <action name="cutAct">
   <property name="icon">
    <iconset resource="scribble.qrc">
     <normaloff>:/icons/cut.png</normaloff>:/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut the selected area</string>
   </property>
  </action>
  <action name="copyAct">
   <property name="icon">
    <iconset resource="scribble.qrc">
     <normaloff>:/icons/copy.png</normaloff>:/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;opy</string>
   </property>
   <property name="toolTip">
    <string>Copy the selected area</string>
   </property>
  </action>
  <action name="pasteAct">
   <property name="icon">
    <iconset resource="scribble.qrc">
     <normaloff>:/icons/paste.png</normaloff>:/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste the selected area</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="scribble.qrc"/>
 </resources>
 <connections/>
</ui>
